SUMMARY.md

```diff
@@ -1,17 +1,17 @@
 # Table of contents
 
-* [Introduction](README.md)
-* [Staking](staking.md)
-* [Bonds](bonds.md)
-* [Policy](policy.md)
-* [Initial Network State](initial-network-state.md)
-* [Market Dynamics](market-dynamics.md)
-* [Architecture](architecture.md)
-* [Equations](equations.md)
-* [Translations](translations.md)
+- [Introduction](README.md)
```

Comment:
This was my editor. This is apparently the default or preferred formatting for markdown. Not sure if this is right though. Let me know if this is a problem.